web/src/pm_list_data.js

```diff
@@ -12,28 +12,37 @@ const max_conversations_to_show = 8;
 // Maximum number of conversation threads to show in default view with unreads.
 const max_conversations_to_show_with_unreads = 15;
 
-export function get_active_user_ids_string() {
+export function get_active_user_ids() {
     const filter = narrow_state.filter();
 
     if (!filter) {
-        return undefined;
+        return {
+            user_ids_string: undefined,
+            user_ids_array: undefined,
+        };
     }
 
     const emails = filter.operands("dm")[0];
 
     if (!emails) {
-        return undefined;
+        return {
+            user_ids_string: undefined,
+            user_ids_array: undefined,
+        };
     }
 
-    return people.emails_strings_to_user_ids_string(emails);
+    return {
+        user_ids_string: people.emails_strings_to_user_ids_string(emails),
+        user_ids_array: people.emails_strings_to_user_ids_array(emails),
+    };
```

Comment:
I don't love this object API. I think it'd be better to just have a function with this name return a `user_ids_array`, and have code that wants a `user_ids_string` out of it do the conversion via sorting+joining (we don't have a method for this, but it'd be easy to add a `people.ids_array_to_user_ids_string`), or `user_ids_string_to_ids_array` for the opposite direction.